attachments.md

```diff
@@ -5,21 +5,21 @@
 
 ## Introduction
 
-Matroska supports storage of related files and data in the Attachments Top-Level Element. Attachments can be used to store related cover art, font files, transcripts, reports, error recovery files, picture or text-based annotations, copies of specifications, or other ancilliary files related to the Segment.
+Matroska supports storage of related files and data in the `Attachments Element` (a `Top-Level Element`). Attachments can be used to store related cover art, font files, transcripts, reports, error recovery files, picture or text-based annotations, copies of specifications, or other ancillary files related to the `Segment`.
 
```

Comment:
> Attachments can be used…

Could probably be "`Attachments` can be used…" as we're talking about the actual elements. So far we haven't defined what the feature "attachment" is, only what the `Attachment Element` can do. Yes, it's a philosophical distinction. So sue me! 😁